index.html

```diff
@@ -70,6 +76,255 @@ <h1>I'm Abdi</h1></div>
+                                        <li> <a href="#">  <button type="button" id="button_link">Ruby</button></a></li>
+                        </ul>
+                </div>
+                <div class="seeProjectCards">
+                    <a href="#"><button type="button" id="button_linkProj">See Project</button></a>  
+              </div>
+            
+              
+            
+            </div>
+
+      
+            <div class="cards card7">
+                <h3 class="cards-header">Profesional Art <br>Printing Data</h3>
+                <div class="cards_description">
+                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's 
+                        standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+                </div>
+              
+                <div class="card_buttons">
+                    <ul>
+                        <li> <a href="#">  <button type="button" id="button_link">html</button></a></li>
+                                        <li>  <a href="#">   <button type="button" id="button_link">bootstrap</button></a>
+                                        </li>
+                                        <li> <a href="#">  <button type="button" id="button_link">Ruby</button></a></li>
+                        </ul>
+                </div>
+                <div class="seeProjectCards">
+                    <a href="#"><button type="button" id="button_linkProj">See Project</button></a>  
+              </div>
+            
+              
+            
+            </div>
+           
+
+
+        </div>
+       
+       
+        
+
+
+
+
+
+
+
+    </section>
+    <div class="about_Me">
+        <div class="abtMe_content">
+        <h2 class="abtMe_Header">About me
+            
+        </h2>
+        <div class="abtMedescription">
+            <p>Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project 
+                you need coded, don’t hestiate to contact me.</p>
+        </div>
+        <button type="button" class="myresumeButton">Get My Resume</button>
+        <img src="./assets/images/lineForAbtMe.png" id="lineAbtMe">
+
+
+
+
+
+
+        </div>
+
+
+
+
+
+
+
+
+        <div class="experiance">
+            <ul>
+                <li>
+                   <div class="title_container">  <img src="./assets/images/rombusIcon.png" class="title_image"> <h3> Languages<h3></h3></div>
+                  
+                  <ul>
+                    <li>JavaScript</li>
+                    <li>HTML</li>
+                    <li>CSS</li>
+                  </ul>
+                </li>
+                  <li>
+                    <div class="title_container">  <img src="./assets/images/RectangleIcon.png" class="title_image"> <h3> Frameworks<h3></h3></div>
```

Comment:
[OPTIONAL] Organizing your code will make it look more proffessional and readable. :1234:  I know it can be tough when you have a deadline to beat and working under stress, but trust me, your future self will be very grateful if you start using the best code convention practices and organised your code well. :stuck_out_tongue_winking_eye:  Kindly check this [link](https://www.freecodecamp.org/news/how-to-indent-in-html-and-why-it-is-important/) at freeCodeCamp for more information and inspiration.